react-props-and-expressions/my-app/src/App.jsx

```diff
@@ -0,0 +1,39 @@
+import { useState } from 'react';
+import reactLogo from './assets/react.svg';
+import viteLogo from '/vite.svg';
+import './App.css';
+import CustomButton from './CustomButton';
+import './CustomButton.css';
```

Comment:
This should be imported into your `CustomButton` file. Since these styles only apply there, we don't want to import them here